swtbot_test/src/parameters/ProjectParameters.java

```diff
@@ -353,12 +353,12 @@ public class AmazonConfig{
 	}
 	
 	public class FileLocation{
-		public static final String AZURE_RTOS_LOCATION = "D:\\rtos_package\\Azure\\6.2.1-rx-2.0.0";
```

Comment:
E: These locations should be the default location so that it can run on any machine without changing to the location you define on your machine.

Reply:
As I remember, I have created a powerpoint file, to instruct users carefully how to changed these locations base on each machine when anyone want to run these swtbot script. I can also make FitModules location and RTOS location to default, but other locations, such as locations to test for import configuration, or empty folder location, cannot be set default to anywhere, and however the case user have to config these location themself anyway. There fore, these locations, In my opinion, can be ignore in this pull request